EnglishFilter/sections/general_extensions.txt

```diff
@@ -3972,7 +3972,8 @@ neowin.net#$#div[class^="banner-"] { height: 0!important; min-height: 0!importan
 !-------------------
 !
 ! https://github.com/AdguardTeam/AdguardFilters/pull/73469#issuecomment-769622353
-shortlinkpay.com$$script[tag-content="zoneSett"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/73712
+usersdrive.com,shortlinkpay.com$$script[tag-content="zoneSett"]
```

Comment:
Ok, let's try it. Remove domains from the rule.
If something goes wrong, I'll revert it.

Reply:
Yes, Thanks for keeping my suggestion.

Reply:
I will also inform you if anything goes wrong.